backend/src/repositories/reportRepository.js

```
@@ -0,0 +1,59 @@
+      WHERE dr."disasterDisasterId" = $1`, [disasterId]
+    );
+
+    // Query for all teams associated with the disaster
+    const [teams] = await reportRepository.query(
+      `SELECT DISTINCT t.team_id, t.name, t.responsibility
+      FROM teams t
+      WHERE t."disasterDisasterId" = $1`, [disasterId]
+    );
+
+    return {
+      totalVolunteers: parseInt(stats.totalVolunteers || 0, 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)